src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1280,6 +1280,7 @@ internal void OnPresenterScrolled(double horizontalOffset, double verticalOffset
 			if (isIntermediate && UpdatesMode != Uno.UI.Xaml.Controls.ScrollViewerUpdatesMode.Synchronous)
 			{
 				RequestUpdate();
+				_snapPointsTimer?.Stop();
```

Comment:
Are we able to add some tests for this ?

Reply:
A manual test is added.

Reply:
added an manual test

Reply:
while possible with runtime test, it is just too unreliable:
// as in not failing often enough with the fix disabled
// problem being step@1 doesnt trigger a scroll most of time
// it is a guarantee repro when it does

<details>
<summary>src</summary>

```cs
// for runtime adjustment with watch window in vs
public static int D1 = 250;
public static int D2 = 100;
public static int R = 5;

[TestMethod]
public async Task When_Quickly_Scrolling_Twice_qweqwe()
{
    FeatureConfiguration.Asd.UnoSnapActivationDelay = TimeSpan.FromMilliseconds(D1);
    FeatureConfiguration.Asd.UnoIosSnapRubberBandFix = false;

    var lv = new ListView
    {
        IsSwipeEnabled = true,
        ItemsSource = new[] { 0, 1, 2 },
        ItemsPanel = (ItemsPanelTemplate)XamlReader.Load("""
            <ItemsPanelTemplate xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
                <ItemsStackPanel Orientation="Horizontal" />
            </ItemsPanelTemplate>
            """),
        ItemTemplate = (DataTemplate)XamlReader.Load("""
            <DataTemplate xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
                <Border Padding="5" Width="90" Height="90" Background="Pink">
                    <TextBlock Text="{Binding}" HorizontalAlignment="Center" />
                </Border>
            </DataTemplate>
            """),
        ItemContainerStyle = NoSpaceContainerStyle,
    };
    var container = new Grid
    {
        Width = 100,
        Height = 100,
        Children = { lv },
    };

    WindowHelper.WindowContent = container;
    await WindowHelper.WaitForLoaded(container);

    var SUT = lv.FindFirstDescendant<ScrollViewer>();
    SUT.HorizontalScrollBarVisibility = ScrollBarVisibility.Visible;
    SUT.VerticalScrollBarVisibility = ScrollBarVisibility.Disabled;
    SUT.HorizontalScrollMode = ScrollMode.Enabled;
    SUT.VerticalScrollMode = ScrollMode.Disabled;
    SUT.IsHorizontalRailEnabled = true;
    SUT.IsVerticalRailEnabled = false;
    SUT.HorizontalSnapPointsType = SnapPointsType.MandatorySingle;
    SUT.HorizontalSnapPointsAlignment = SnapPointsAlignment.Center;

    await WindowHelper.WaitForIdle();

    var lvi0 = (ListViewItem)lv.ContainerFromIndex(0);
    //var unotree = container.ShowLocalVisualTree(0);
    //var tree = container.TreeGraph();

    for (int i = 0; i < R; i++)
    {
        // scroll to last item#2, wait for scrolling+snapping to fully complete
        SUT.ChangeView(SUT.ScrollableWidth, null, null);
        await Task.Delay(1000);

        // overscroll by 10px, so it should still snap back to last item
        // BUT before the snapping occurs, quickly scroll to the left by a bit
        SUT.ChangeView(SUT.ScrollableWidth + 10, null, null); // step@1
        await Task.Delay(D2);
        SUT.ChangeView(SUT.ScrollableWidth - lvi0.ActualWidth / 2, null, null); // step@2
        await Task.Delay(1000);
        await WindowHelper.WaitForIdle();

        // the expected end result of all above steps is to snap to item#1
        // but due to uno#10892, the snapping would trigger with the offset of step@1
        // and snap to item#2 instead...

        // abs distance to 2nd item center is 1.5*item_width
        // then substract by viewport-width/2 to align with the center of viewport (without this, it would align to the left instead)
        Assert.AreEqual(1.5 * lvi0.ActualWidth - SUT.ViewportWidth / 2, SUT.HorizontalOffset);
    }
}
```
</details>